tests/e2e/ansible/install_docker.yml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Red Hat
```

Comment:
nit: For all new code, let's follow CNCF [guidelines](https://github.com/cncf/foundation/blob/main/copyright-notices.md) 

and use something like this:
```
Copyright Confidential Containers Contributors
```

We need to do this for all the code, but let's start with this one :-)

Reply:
@bpradipt I read that before but I misunderstood the doc. thanks for the clarification, I will fix them on this PR.